packages/helm-charts/blockscout/values-alfajores.yaml

```diff
@@ -50,3 +50,5 @@ blockscout:
     recaptchaSecretName: alfajores-blockscout-recaptcha
   metrics:
     enabled: true
+  secrets:
+    grafanaCloud: gcp:secretmanager:projects/365081401101/secrets/alfajores-blockscout-grafana-cloud
```

Comment:
Let's stick to the `%network%-blockscout-%camelCasedSecretKey%` notation, like in all other cases.